src/fontra/client/web-components/designspace-location.js

```diff
@@ -7,7 +7,7 @@ export class DesignspaceLocation extends UnlitElement {
   static styles = `
     :host {
       display: grid;
-      grid-template-columns: 25% auto;
+      grid-template-columns: max-content auto;
```

Comment:
Could you try fixing the slider resizing while keeping the first column at 25%? The reason it was 25% is to prevent very long axis names to squish away the sliders too much.